content/courses/onchain-development/intro-to-anchor.md

```diff
@@ -17,6 +17,54 @@ description: "Create your first Solana onchain program in Anchor."
   incoming instructions, and checking the right accounts are provided - so you
   can focus on building your Solana program.
 
+## Overview
+
+Anchor is a framework designed to simplify the process of writing Solana
+programs. By **abstracting away much of the low-level details**, Anchor enables
+developers to focus more on the logic of their decentralized applications
+(dApps) without worrying about the complexities of Solana's native APIs.
+
+Anchor provides many features that make building Solana programs easier and
+safer:
+
+- `Pre-built Security Mechanisms`: Anchor implements common security checks
+  automatically, reducing the chances of errors.
+- `Automatic Account Validation`: Anchor validates accounts by checking for:
+- `Type correctness`: Ensures that accounts match the expected data types.
+- `Uniqueness`: Ensures that each account is correctly initialized and doesn't
+  conflict with existing ones.
+- `Simplified Data Handling`: Anchor automatically serializes and deserializes
+  the data passed in transactions, making it easier to manage the inputs and
+  outputs of your program.
+- `Instruction Routing`: Anchor routes incoming instructions to the appropriate
+  function, simplifying the process of executing program logic.
+
+Here’s how Anchor works in practice:
+
+Programs are deployed at addresses In the same way, Solana programs can be
+deployed using their program ID, and Anchor programs can also be deployed with a
+unique public key. This keypair is created during the initialization of an
+Anchor project and is stored in the target/deploy directory.
```

Comment:
'Programs are deployed at addresses' needs a full stop.

'In the same way, Solana programs can be deployed using their program ID,' - in the same way as what? I thought the previous sentence referred to Solana programs.

'and Anchor programs can also be deployed with a unique public key' - Anchor programs are also Solana programs.